gallery/src/components/NxTabs/NxTabsPage.tsx

```diff
@@ -74,6 +74,32 @@ export default function NxTabsPage() {
                   Called with the index of the newly selected tab when the currently selected tab changes.
                 </NxTableCell>
               </NxTableRow>
+              <NxTableRow>
+                <NxTableCell>
+                  <NxCode>&lt;div&gt;</NxCode>, <NxCode>&lt;ul&gt;</NxCode>, <NxCode>&lt;li&gt;</NxCode> HTML Attributes
+                </NxTableCell>
+                <NxTableCell>
+                  <a target="_blank"
+                     rel="noopener"
+                     href="https://developer.mozilla.org/en/docs/Web/HTML/Element/div">
+                    HTML div Attributes
+                  </a><br/>
+                  <a target="_blank"
+                     rel="noopener"
+                     href="https://developer.mozilla.org/en/docs/Web/HTML/Element/ul">
+                    HTML ul Attributes
+                  </a><br/>
+                  <a target="_blank"
+                     rel="noopener"
+                     href="https://developer.mozilla.org/en/docs/Web/HTML/Element/li">
+                    HTML li Attributes
+                  </a>
```

Comment:
This should be split out to the doc sections for the various subcomponents

Reply:
> This should be split out to the doc sections for the various subcomponents

Done here: https://github.com/sonatype/sonatype-react-shared-components/pull/397/commits/574d10f056bb5fda1e32c77dc97c6f9771022ba7